src/services/beam.service.ts

```diff
@@ -36,67 +36,94 @@ export class BeamService {
         return ((await queryBuilder.getRawOne()) as any).sum;
     };
 
-    sumTotalAmountInRange = async (startAt: Date, endAt: Date, monthly = false): Promise<{ key: string; value: number }[]> => {
+    sumTotalAmountInRange = async (startAt: Date, endAt: Date, groupType: string): Promise<{ key: string; value: number }[]> => {
         const query = await this._repository.query(`
-            with dates as (
```

Comment:
Make sure to check presence of optional groupType property and maybe, assign a default behavior?